tests/unit/ComputeRecordCompleteness.unitTest.ts

```diff
@@ -148,21 +148,21 @@ describe("Compute Record Completeness", () => {
       it("should return TESTABLE if all core mandatory are completed and one of the non-core mandatory attributes is missing", () => {
         const record: any = cloneDeep(mockData[126]);
         delete record.techRecord[0].dimensions.length;
-        const recordCompleteness = computeRecordCompleteness(record, record.trailerId);
+        const recordCompleteness = computeRecordCompleteness(record);
         expect(recordCompleteness).toEqual(RECORD_COMPLETENESS_ENUM.TESTABLE);
       });
 
-      it("should return TESTABLE if all core mandatory are completed and primaryVrm is missing", () => {
+      it("should return COMPLETE if all core mandatory are completed and primaryVrm is missing", () => {
         const record: any = cloneDeep(mockData[126]);
         delete record.primaryVrm;
-        const recordCompleteness = computeRecordCompleteness(record, record.trailerId);
-        expect(recordCompleteness).toEqual(RECORD_COMPLETENESS_ENUM.TESTABLE);
+        const recordCompleteness = computeRecordCompleteness(record);
```

Comment:
I do worry about the impacts of removing primaryVrm as a required attribute after it broke this test, might be one to ask VTA team

Reply:
We should push forward for the correct logic. On VTM, we never expect trailers to have primary VRMs. If it causes problems, I can address them then or revert this.